### ISSUE TITLE
Codegen is still broken for associative tables

### ISSUE THREAD

**Issue by acsellers** [stephenafamo/bob#52]
Thank you for the changes on Issue #50 , but the changes expose a new issue. usersRoles[i] is nil, so assigning RoleID and UserID cause a panic due to the nil pointer dereference. I have to add `usersRoles[i] = &UsersRoleSetter{}` as the first line in the for loop to fix the nil assignment issue.

```
	usersRoles := make([]*UsersRoleSetter, len(related))

	for i, rel := range related {
		usersRole := usersRoles[i]
		usersRole.UserID = omit.From(o.ID)
		usersRole.RoleID = omit.From(rel.ID)
	}
```

**Comment by stephenafamo** [comment]
Try 8b1005 and see if it is fixed there

**Comment by stephenafamo** [comment]
Released in `v0.21.0`
